cromwell-helm/templates/cromwell-config.yaml

```diff
@@ -79,6 +79,12 @@ data:
                 project = "{{ .Values.config.gcsProject }}"
               }
             }
+
+            virtual-private-cloud {
+              network-label-key = "vpc-network-name"
+              subnetwork-label-key = "vpc-subnetwork-name"
+              auth = "application-default"
+            }
```

Comment:
What do these lines of code do? Why do we need them?

Reply:
Explained this in mob

Reply:
Saloni's answering of this question:

Cromwell no longer runs jobs in default networks. Now, Terra has the option to create your own network so you're not limited by the IP. So we running jobs in this particular **private** network (to be supplied) rather than the default network.